### PR TITLE
Allow to switch between ramp and category view

### PR DESCRIPTION
**Pull request by javierarce**
animation

### CODE REVIEW

File: lib/assets/test/spec/cartodb3/components/form-components/editors/fill/input-color/input-categories/input-color-categories.spec.js [hunk 1]
```diff
@@ -36,9 +44,9 @@ describe('components/form-components/editors/fill/input-color/input-ramps/input-
   });
 
   it('should show the loader', function () {
+    spyOn(this.view, '_startLoader');
```
Comment (javierarce) [inline lib/assets/test/spec/cartodb3/components/form-components/editors/fill/input-color/input-categories/input-color-categories.spec.js]: Sorry, I couldn't find a better way to test this, and the old way wasn't ok anymore.

**Comment by javierarce** [comment]
CR: @xavijam 🙏

File: lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-color/input-ramps/input-color-ramps.js [hunk 2]
```diff
@@ -76,6 +81,17 @@ module.exports = CoreView.extend({
     });
   },
 
+  _createTooltip: function () {
+    this._helpTooltip = new TipsyTooltipView({
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-color/input-ramps/input-color-ramps.js]: We should add `this.addView(this._helpTooltip);`, shouldn't we?

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/form-components/editors/fill/input-color/input-ramps/input-color-ramps.js]: Ahhh we clean it when we remove the view, 👍

**Comment by xavijam** [comment]
I feel I can baptize this component as the most complicated component of the builder. Well done man!
🚢 🇮🇹

**Pull request merged**